assets/sass/modules/content-blocks/_global.scss

```diff
@@ -25,6 +26,27 @@
 		} // &.container-width.
 	} // &.has-background
 
+	// Remove the extra padding on nested no-background blocks.
+	&.no-background,
```

Comment:
Can we combine these two declarations + the one on line 41? It looks like they're all targeting the same children, so we could possibly save a few lines there if it doesn't cause any unexpected issues.

Reply:
_backs away_